style.css

```diff
@@ -866,20 +872,23 @@ input:focus:invalid {
 
   #skills {
     flex-direction: row;
-    justify-content: space-around;
+    justify-content: space-between;
     align-items: baseline;
-    padding: 6rem 0;
+    padding: 4rem 0;
     margin-left: -5rem;
+   
  }
 
   #skills h3 {
-    margin-bottom: 2rem;
+    font-size: 1.2rem;
   }
 
+
   #languages-list,
   #skills-list,
   #frameworks-list {
-    align-items: flex-start;
+   align-items:flex-start
+       
```

Comment:
- [ ]  As requested by the previous reviewer you should stick to the design as much as possible like color, font, and spacing. please make sure that you adjust the spacing as in the design. Please see the image below for reference. Also, adjust the size of the list.
- [ ] Your design 
![image](https://github.com/Tshepo45/Portfolio-desktop-version/assets/68823331/51ccab4c-210b-4341-a993-3d5dfe8da8f0)

- [ ] Figma Design 
![image](https://github.com/Tshepo45/Portfolio-desktop-version/assets/68823331/cb60d81b-9ebe-45cf-99d5-7b03101cdc2f)